### PR TITLE
Want to merge?

### PR DESCRIPTION
**Pull request by RobbieClark**

### CODE REVIEW

File: protocol/protocol.Rmd [hunk 1]
```diff
@@ -198,15 +198,16 @@ pander(interviewees, split.table = Inf, justify = "left")
...
+To answer RQ1 we will ask all respondents to tell us about their experiences of partnership models at all stages relevant to them. 
+Specifically we plan on asking them about:
 
 * the partnership's potential improvements and the feasibility of these
 * the partnership's strengths
 * the partnership's weaknesses 
 * the partnership's impact on research quality (as the interviewee defines quality)
 * the partnership's impact on the efficiency of research process (i.e. from idea conception to publication)
 * their reasons for involvement with the partnership
+* their desire to continue or repeat involvement
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: I think this simply is a way of getting at what people like/don't like about the partnerships

File: protocol/protocol.Rmd [hunk 2]
```diff
@@ -141,9 +141,9 @@ They are also preferable to focus groups because focus groups are typically more
 
 ## Respondents and recruitment
 
-To understand the experiences of all stakeholder groups in funder-RR partnerships we will need to recruit **[number?]** people from each of the four stakeholder groups, (1) authors of partnership submissions, (2) personnel at partner funders, (3) editors at partner journals, and (4) reviewers of partnership submissions. 
+To understand the range of experiences in each of the funder-journal partnerships we will need to recruit at least one person from each of the four stakeholder groups: (1) authors of partnership submissions, (2) personnel at partner funders, (3) editors at partner journals, and (4) reviewers of partnership submissions.
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: one person ... from each of the 6 partnerships?

File: protocol/protocol.Rmd [hunk 3]
```diff
@@ -293,20 +294,26 @@ Once consent is received the respondent and interviewer will arrange the intervi
 Interviews will either be face-to-face in Bristol, or remotely using Skype or telephone, whichever the respondent prefers. 
 However, travel will not be reimbursed.
 It may be justifiable for the interviewer to offer to travel to meet the interviewee.
-JT, KD and RC will conduct the interviews, they will be audio recorded and last around **[90?** minutes.
-If the audio recording is of poor quality the respondent will be invited for a second interview.
+JT, KD and RC will conduct the interviews, which will be audio recorded and last around 60 minutes.
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: 90 minutes feels better to allow for set up time

File: protocol/protocol.Rmd [hunk 4]
```diff
@@ -293,20 +294,26 @@ Once consent is received the respondent and interviewer will arrange the intervi
...
+If the audio recording is of poor quality the respondent might be invited for a second interview.
 RC will transcribe the interviews.
-The transcripts will be sent to the respondent to allow the to confirm its accuracy.
+**The transcripts will be sent to the respondent to allow the to confirm its accuracy**. [[<-- this may or may not happen, I remember Anna warning us against defaulting to this in her talk at Cumberland Lodge. We should collectively think and decide on this before putting a strong statement like this in (#robbie)]]
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: From what I remember Anna was warning us against sending the analysis results to ppts - I think we definitely need to do this to ensure people are happy sharing their data and will help reduce inaccuracies

File: protocol/protocol.Rmd [hunk 5]
```diff
@@ -322,23 +329,23 @@ There are no expected hazards, due to the study consisting of an interview of no
 
 All aspects of the Data Protection Act will be adhered to. 
 Respondent contact details, audio recordings and any respondent identifier logs will be stored on password protected university computers and kept confidential. 
-These will be kept for one year after study completion or until data are made open (whichever comes first), after which these documents will be destroyed **sure we want to destroy recordings?**
+These will be kept for one year after study completion or until data are made open (whichever comes first), after which these documents will be destroyed **sure we want to destroy recordings? (I think it's common thing to do once satisfactory transcriptions have been made @rob)**
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: OK i'll check with Anna

File: protocol/protocol.Rmd [hunk 6]
```diff
@@ -322,23 +329,23 @@ There are no expected hazards, due to the study consisting of an interview of no
...
-In accordance with the FAIR (Findable, Accessible, Interoperable, Reusable) principles [@wilkinson2016], all anonymised data will be uploaded to the University of Bristol's data.bris research data repository (https://data.bris.ac.uk/data/) at the end of the study for long-term archiving. 
+In accordance with the FAIR (Findable, Accessible, Interoperable, Reusable) principles [@wilkinson2016], all anonymised data will be uploaded to the University of Bristol's Research Data Repository (https://data.bris.ac.uk/data/) at the end of the study for long-term archiving. 
 It will be uploaded under the most stringent category of 'Controlled data'.
 Therefore to access the data, bona fide researchers will need to secure a Data Access Agreement from their host institution. 
-With their host institution’s approval, a request for access will be judged by the repository’s Data Access Committee.
+With their host institution’s approval, a request for access will be judged by the repository’s Data Access Committee, who will confer with either RC, KD or JT to confirm that the planned research can be supported by the dataset.
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: I think we need to pick one of us no?

File: protocol/protocol.Rmd [hunk 7]
```diff
@@ -322,23 +329,23 @@ There are no expected hazards, due to the study consisting of an interview of no
...
 More information about Controlled Data access requests is available at https://www.bristol.ac.uk/staff/researchers/data/accessing-research-data/.
 Study data will be kept for a minimum of 15  years. 
-Anonymised extracts from the transcripts may be used in any publications resulting from the research.
+Anonymised extracts from the transcripts may be used in any publications, conference or teaching materials resulting from the research.
 
 ## Revoked data
 
-If a respondent decides that they do not want their data used during the interview or after their participation they have the right to request for the interview to be ended and/or for their data to be withdrawn. 
-They can request this up to one year after study completion or until the data are made open (whichever comes first). 
+If, during or after the interview, a respondent decides that they do not want their data to be used, they have the right to request for the interview to be ended and/or for their data to be withdrawn.
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: This doesn't make much sense that they can 'end the interview' after the interview

File: protocol/protocol.Rmd [hunk 8]
```diff
@@ -418,7 +425,7 @@ Children's Tumor Foundation (CTF) & PLoS ONE: Drug Discovery Initiative - Regist
     - This is done by appointment over the phone to CTF-DDIRR director, Salvo, and transcribed-- cuts down on turnaround waiting for reviews 
 
 - CTF decides on basis of reviews which LOIs to invite for full submission 
-- Researchers make full submission in Stage 1 RR format; this is submitted to PLoS One, along with original CTF reviews, for review (including statistical review) by PLoS ONE reviewers 
+- Researchers make full submission in stage 1 RR format; this is submitted to PLoS One, along with original CTF reviews, for review (including statistical review) by PLoS ONE reviewers
```
Comment (katiedrax) [inline protocol/protocol.Rmd]: haha looks like you can't decide if you want stages to be Stage or stage ;-)

File: protocol/protocol.Rmd (hunk 2 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: One person per stakeholder group per partnership

File: protocol/protocol.Rmd (hunk 1 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: Agree, but we do have it as a question. Can remove if you think unneccessary

File: protocol/protocol.Rmd (hunk 3 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: Sure, but some might only be 30mins. Perhaps say 45-90 (contingent on participant availability?)

File: protocol/protocol.Rmd (hunk 4 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: I'm 98% it's other way around
- If you send people transcripts they might start asking to edit individual bits that they have mentioned (*"I can't really have said that!?"*) and it's just generally very time consuming and not really worthwhile, it's hardly like they will remember everything they said!
- Sending final analysis is a form of triangulation ("member checking" [see my notes on Qual Research]) as it allows participants to judge whether your analysis is a fair reflection of what they said/meant

File: protocol/protocol.Rmd (hunk 6 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: Yeh probs at some point we need to have a specific member for correspondence. The way I've phrased it is technically not incorrect though.

File: protocol/protocol.Rmd (hunk 7 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: and/or (as appropriate) was what I had in mind. Syntactically seems same as the previous edition, which I thought was quite wordy and needed breaking down a bit.

File: protocol/protocol.Rmd (hunk 8 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: decided on stage!!!!!!

File: protocol/protocol.Rmd (hunk 3 above)
Comment (katiedrax) [reply protocol/protocol.Rmd]: mmm dunno - we could say we will ask participants for 90 minutes and if their availability cannot allow it we will do as long as they can?

File: protocol/protocol.Rmd (hunk 4 above)
Comment (katiedrax) [reply protocol/protocol.Rmd]: Yep member checking we are definitely not doing (b/c data identifiable etc). OK lets not

File: protocol/protocol.Rmd (hunk 6 above)
Comment (katiedrax) [reply protocol/protocol.Rmd]: We can decide on this togther

File: protocol/protocol.Rmd (hunk 3 above)
Comment (RobbieClark) [reply protocol/protocol.Rmd]: JT, KD and RC will conduct the interviews, they will be audio recorded and last around 90 minutes, subject to participants' availability.

File: protocol/protocol.Rmd (hunk 6 above)
Comment (katiedrax) [inline protocol/protocol.Rmd]:
```suggestion
With their host institution’s approval, a request for access will be judged by the repository’s Data Access Committee, who will confer with **[tbd RC, KD or JT] to confirm that the planned research can be supported by the dataset.
```

File: protocol/protocol.Rmd (hunk 5 above)
Comment (katiedrax) [inline protocol/protocol.Rmd]:
```suggestion
These will be kept for one year after study completion or until data are made open (whichever comes first), after which these documents will be destroyed **[check when to destroy recordings Anna]**
```

File: protocol/protocol.Rmd (hunk 4 above)
Comment (katiedrax) [inline protocol/protocol.Rmd]:
```suggestion
```

Comment (RobbieClark) [reply protocol/protocol.Rmd]: I don't get why we wouldn't do member checking - it's just like getting them to read a pre-print (either way it's going to be published!)

File: protocol/protocol.Rmd (hunk 1 above)
Comment (katiedrax) [inline protocol/protocol.Rmd]:
```suggestion
```

File: protocol/protocol.Rmd (hunk 2 above)
Comment (katiedrax) [inline protocol/protocol.Rmd]:
```suggestion
To understand the range of experiences in each of the funder-journal partnerships we will need to recruit at least one person from each model and from each of the four stakeholder groups: (1) authors of partnership submissions, (2) personnel at partner funders, (3) editors at partner journals, and (4) reviewers of partnership submissions. 
```

File: protocol/protocol.Rmd (hunk 3 above)
Comment (katiedrax) [reply protocol/protocol.Rmd]: Nice

Comment (katiedrax) [inline protocol/protocol.Rmd]:
```suggestion
JT, KD and RC will conduct the interviews, they will be audio recorded and last around 90 minutes, subject to participants' availability.
```

**Comment by RobbieClark** [comment]
Go team!

**Comment by RobbieClark** [comment]
Nice work! How's the lit review/ intro kraken?

**Comment by katiedrax** [comment]
haha mate you closed the pull request with pulling it!

**Comment by RobbieClark** [comment]
I just noticed that a lot of changes weren't into effect!

**Pull request merged**